docs/root/configuration/http/http_conn_man/stats.rst

```diff
@@ -101,12 +106,24 @@ following statistics:
    downstream_rq_4xx, Counter, Total 4xx responses
    downstream_rq_5xx, Counter, Total 5xx responses
 
+Http3 per listener statistics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Http3 statistics with the form of http3.<stat_prefix>.*:
+
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 1, 1, 2
+
+   [direction].[source].quic_connection_close_error_code_[error_code], Counter, A collection of counters that are lazily initialized to record each quic connection close error code that's present. direction could be *upstream* or *downstream*. source could be *self* or *peer*.
```

Comment:
If we're doccing up listener stats, won't direction always be downstream?

And actually I hate to catch this now, but probably rather than self/peer it should be tx/rx for consistency with other Envoy stats.  I find self and peer more intuitive (maybe 'cause that's what I'm used to) but I think most of the other stats (like resets tx_reset/rx_reset) are documented that way with tx for things sent to the peer and rx being codes received from the peer.

Reply:
You are right. The listener stats should be all downstreams.

I haven't figured out the details on upstream stats. So I will update the doc again when I add them.

Done changing names to tx/rx.